code/game/objects/structures/door_assembly.dm

```diff
@@ -11,6 +11,7 @@
 	var/mineral = null
 	var/obj/item/electronics/airlock/electronics = null
 	var/airlock_type = /obj/machinery/door/airlock //the type path of the airlock once completed
+	var/multi_tile = FALSE
```

Comment:
you can dmdoc it like this

```suggestion
	var/multi_tile = FALSE //! DMDoc goes here
```

just so you don't have it looking weird where it is while still maintaining that statistic/documentation.